### PR TITLE
[fix #6693] Add Valentine’s Day fundraising banner

### LINKED ISSUE #6693: Fundraising banner for February/Valentine campaign
| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | |
| ENG | |
| SEO | |
| ANA | |
| DES | |
| COPY | |

MoFo is planning a mid-February fundraising campaign and wants to bring back the home page banner for a brief time, with design and messaging relating to Valentine's Day. It will be English-only. 

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Launch on Friday Feb 8th
- [ ] Switch

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] 

#### Tasks
- [ ] Code it
- [ ] Code switch
- [ ]

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adds a fundraising banner for the upcoming Valentine's Day campaign, due to go live on Friday 8 February and come down early on Friday 15 February (the morning after Valentine's Day).

Uses the switch `fundraising-feb2019` and is English-only, no l10n required.

## Issue / Bugzilla link
#6693 

## Testing
https://bedrock-demo-craigcook.oregon-b.moz.works/en-US/

Dismissing the banner sets a 2 day cookie called `fundraiser-feb2019` so you'll need to delete that cookie to see the banner again if you're testing.

### CODE REVIEW

File: media/css/mozorg/home/fundraiser-feb2019.scss [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    }
+
+    // hide by default if JS is available to avoid flicker
+    // (if visitor previously dismissed)
+    .js & {
+        display: none;
+    }
+
+    .mzp-c-button.mzp-t-dark {
+        @include border-box;
```
Comment (amychurchwell) [inline media/css/mozorg/home/fundraiser-feb2019.scss]: Lines 28-39 are not needed anymore. :+1:

File: media/css/mozorg/home/fundraiser-feb2019.scss [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        cursor: pointer;
+        display: inline-block;
+        font-weight: bold;
+        padding: ($padding-md - 2px) ($padding-xl - 2px); //2px extra padding removed to compensate for 2px border.
+        text-decoration: none;
+
+        &:focus,
+        &:hover {
+            @include transform(translate(-2px, -2px));
+            background-color: rgba(255, 255, 255, .15);
```
Comment (amychurchwell) [inline media/css/mozorg/home/fundraiser-feb2019.scss]: only property needed for focus & hover is `background-color`

File: media/css/mozorg/home/fundraiser-feb2019.scss [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            box-shadow: 4px 4px 0 0 $color-white;
+            color: $color-white;
+        }
+
+        &.mzp-t-secondary {
```
Comment (amychurchwell) [inline media/css/mozorg/home/fundraiser-feb2019.scss]: No css required for `.mzp-t-secondary` anymore.

File: media/css/mozorg/home/fundraiser-feb2019.scss (hunk 2 above)
Comment (amychurchwell) [reply media/css/mozorg/home/fundraiser-feb2019.scss]: `background-color: transparent`?

**Comment by craigcook** [comment]
Cleaned up the leftover button styles. That was recycled code and most of that has since landed in Protocol.

**Pull request merged**